.travis.yml

```diff
@@ -34,4 +31,5 @@ script:
 - nosetests -v TestNXOSDriver:TestGetterNXOSDriver.test_get_snmp_information
 - nosetests -v TestNXOSDriver:TestGetterNXOSDriver.test_get_users
 - nosetests -v TestNXOSDriver:TestGetterNXOSDriver.test_traceroute
+- nosetests -v TestNXOSDriver:TestNXOSDriver_bgp_neighbors
```

Comment:
I needed to run setUp before each test.  The set_bgp_return_data sets some instance state which I needed to erase.  I could call that anyway, clearing any global state.  "setUpClass" probably makes sense if you're connecting with an actual device, since that probably takes a while to run; for mock tests, setUp suffices, but these could probably be combined with something like the following:

```
    @classmethod
    def setUpClass(cls):
        cls.mock = True
        ... snip ...
        cls.device = NXOSDriver(hostname, username, password, timeout=60)
        if not cls.mock:
            cls.device.open()

    def setUp(self):
        if cls.mock:
            cls.fake = FakeNXOSDevice()
            cls.device.device = self.fake
```

I have to have a reference to self.fake available to set mock data with `set_bgp_return_data` as needed for each test.

Minor point: I was also treating these tests a little differently, since I wanted to check return values.